### PR TITLE
8281523: Accessibility: Conversion from string literal loses const qualifier

### PR DESCRIPTION
**Pull request by djelinski**
This patch fixes jdk.accessibility compilation under VisualStudio 2019 with /Zc:strictStrings enabled.

Strict string handling is already enabled in GCC and Clang. With some effort we should be able to enable it in MSVC as well.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8281523](https://bugs.openjdk.java.net/browse/JDK-8281523): Accessibility: Conversion from string literal loses const qualifier

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7380/head:pull/7380` \
`$ git checkout pull/7380`

Update a local copy of the PR: \
`$ git checkout pull/7380` \
`$ git pull https://git.openjdk.java.net/jdk pull/7380/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7380`

View PR using the GUI difftool: \
`$ git pr show -t 7380`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7380.diff">https://git.openjdk.java.net/jdk/pull/7380.diff</a>

</details>

### CODE REVIEW

**Comment by djelinski** [comment]
Hi, can I get a review for this change? 
This is similar to #7379, but in a different subcomponent (Accessibility vs. AWT). I added `const` qualifier to a bunch of function parameters.

Additionally I removed one (unused) function and one (mismatching) function declaration; build passed, tier1 and tier2 green. Verified with both vs2019 and vs2017.

File: src/jdk.accessibility/windows/native/common/AccessBridgeDebug.cpp [hunk 1]
```diff
@@ -144,37 +144,10 @@ char *printError(char *msg) {
         }
     }
 
-    /**
-     * Send Java debugging info to the appropriate place
-     */
```
Comment (prrace) [inline src/jdk.accessibility/windows/native/common/AccessBridgeDebug.cpp:149]: Deleting this macro - without comment !  - seems to be somewhat out of scope here.

Comment (djelinski) [reply src/jdk.accessibility/windows/native/common/AccessBridgeDebug.cpp:149]: Reverted.

I removed this function because its definition (`PrintJavaDebugString2`) did not match the declaration (`PrintJavaDebugString`), so the function wasn't really usable. I'm fine with leaving it as is, or cleaning up separately.

**Review by prrace — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Review by azuev-java — APPROVED** [review]

**Comment by djelinski** [comment]
hi, could you sponsor this? It's already integrated. Thank you!
